README.md

```diff
@@ -20,16 +22,30 @@ More control can be added using the included Disguise RenderStream components:
    * Note: you can enable/disable the exact GameObject properties using the List editor in the Unity Inspector.
 * **To add designer timeline control**, attach a Disguise RenderStream > **Time Control** component to a Playable Director
 
-## Building a RenderStream asset for disguise designer
+## Building a RenderStream asset for Disguise Designer
 
 To use your Unity Project with disguise designer, you build an executable and import it into your designer project. To do this:
 * Ensure Build Settings are set to build a **Windows x86_64** executable
 * Copy the build folder to your **RenderStream Projects** folder
-* In designer, set up a new RenderStream Layer and point the Asset to your built executable.
+* In Designer, set up a new RenderStream Layer and point the Asset to your built executable.
+
+## Optional: Cluster Rendering
+
+If you are running RenderStream in a cluster and you require deterministic rendering, you will also need to install the [Cluster Display](https://github.com/Unity-Technologies/ClusterDisplay) package to synchronize the nodes, using one of these methods:
```

Comment:
Minor: Deterministic could mean different things to different people.  Things could be deterministic yet still not correlate between nodes.  However I'm not sure I have a better choice of word.  What about "correlation between nodes"?